pyang/plugins/uml.py

```diff
@@ -62,6 +62,11 @@ def add_opts(self, optparser):
                                  dest="uml_inline",
                                  default =False,
                                  help="Inline groupings where they are used."),
```

Comment:
Maybe a note could be added to this text to indicate that groupings from specific modules can be omitted from this option through use of the --uml-no-inline-groupings-from option.